README.md

```diff
@@ -29,17 +44,44 @@ kustomize build config/default | kube apply -f -
 With the controller running on your cluster, you need to first provide AWS login
 credentials in the form of a secret to your cluster:
 
+### Step1: Create Base64 Encoded AWS Credentials
+
+Create a text file with the actual AWS credentials
+
+`creds.txt`
+```sh
+AWS_ACCESS_KEY_ID=ASIAUHGDERSWSWPYVM2DDUMMYDUMMY
+AWS_SECRET_ACCESS_KEY=ZBo0Q0lDUMMYNyL5DUMMYDUMMY
+AWS_SESSION_TOKEN=RYRHHVGVNNNDUMMYDUMMYDUMMYDUMMY
+```
+
+And run the following command from Mac command line
+
+```sh
+openssl base64 -in creds.txt -out base64-creds.txt
+```
+
+Copy the output as a string from `base64-creds.txt` and use it in Secret object under `aws.creds`
+
```

Comment:
it could be slightly easier if we create a secrets dir in the repo and then use `kustomize` to generate the secret. something like the following

```
.
└─ config
    └── secrets
        ├── aws.creds
        └── kustomization.yaml
```
for `aws.creds`

```
[default]
aws_access_key_id=<YOUR ACCESS KEY>
aws_secret_access_key=<YOUR SECRET KEY>
aws_session_token=<YOUR SESSION TOKEN>
```

and for `kustomization.yaml`
```yaml
namespace: default
generatorOptions:
 disableNameSuffixHash: true
secretGenerator:
- name: aws-account-creds
  files:
  - aws.creds
```

and we boil it down to users having to run `aws.creds` and running 
```
kustomize build secrets/ | kubectl apply -f -
```

wdyt?

Reply:
Agree, This is more neater way of doing. 
I am also thinking introducing `examples` folder with different patterns might be useful as well so that we can keep the MAIN readme simple

Reply:
alright. I pull in your PR for now, and we can make the above changes in another iteration.

Reply:
I will raise a PR for the above approach as well

Reply:
that'd be great. thanks!